test/_features/gro/validation.feature

```diff
@@ -72,6 +72,9 @@ Feature: Validations
     Then I should see the 'Tell us how you placed your order' error
     Then I check 'how-radio-online'
     Then I click the 'Continue' button
+    Then I should see the 'Enter your COL Number' error
```

Comment:
could add a validation feature test that says something like: 
Then I fill 'online-toggle-text' with 'COL123462020
  Then I click the 'Continue' button
  Then I should see the 'COL number must follow this format: COL123456/2015' error